dist/index.html

```diff
@@ -38,37 +38,37 @@ <h2 class="w-3/5 text-c100 font-bold leading-tighter">
         </p>
       </div>
 
-      <div class="activities__items flex flex-row justify-center -mt-3 px-8">
+      <div class="showcase-row -mt-3 px-8">
         <!--alt is left null on purpose due to the decorative nature of
             images in this section
             https://webaim.org/techniques/alttext/#decorative -->
         <div class="text-center">
           <img class="mx-auto" src="assets/img/activities/01.png" alt="" />
-          <h3 class="text-c100 text-md py-3 font-bold">We Help</h3>
+          <h3 class="showcase-row__heading">We Help</h3>
           <p>
             Lorem ipsum dolor sit amet, consectetuer adipiscing elit.
           </p>
         </div>
 
         <div class="activity ml-10 pl-10 relative text-center">
           <img class="mx-auto" src="assets/img/activities/02.png" alt="" />
```

Comment:
**One possible approach to this challenge is**

- style the `img` tag so that it always has a fixed width and height.

- Use ` border-radius` CSS property to create a rounded circle and customize border color and width.

- Add some style for the `img` tag parent and nearest ancestor to center the content.

![how we work](https://user-images.githubusercontent.com/8650435/79719344-703c0e00-82ef-11ea-8da2-c5357f5ba0e9.PNG)

Reply:
Yup, I was still experimenting with different solutions. I updated it now, take a look at it and let me know what you think